plugins/azure-devops-backend/src/service/router.ts

```diff
@@ -261,7 +261,10 @@ export async function createRouter(
   });
 
   router.get('/all-teams', async (_req, res) => {
-    const allTeams = await pullRequestsDashboardProvider.getAllTeams();
+    const topTeams = _req.query.topTeams
```

Comment:
OK, not sure what the different naming conventions are for to be perfectly honest but I will make it so.

Reply:
It's less a naming convention and more a practice to avoid linting errors. For example:

https://github.com/backstage/backstage/blob/81f4b646c2be71b98413e3ca442327c9456b181e/plugins/azure-devops-backend/src/service/router.ts#L84-L86

`_req` is not used in the above, if you were to remove the `_` you would get a linting error. You also simply can't remove it - the first item is always the request and we want to use the response object. Hope that helps?